app/services/immediate_email_generation_service/batch.rb

```diff
@@ -61,9 +61,8 @@ def subscriptions_to_fulfill_by_subscriber
         end
     end
 
-    def activated_subscribers(subscriber_ids)
-      Subscriber.activated
-                .where(id: subscriber_ids)
+    def subscribers(subscriber_ids)
+      Subscriber.where(id: subscriber_ids)
                 .index_by(&:id)
```

Comment:
Minor, this doesn't look like it needs to be multiline anymore. Arguably this doesn't really need to be a method and could be just an extra line in `subscriptions_to_fulfill_by_subscriber`

Reply:
I've removed the method and updated `subscriptions_to_fulfill_by_subscriber` accordingly.